.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -28,9 +28,29 @@ _Put an `x` in the boxes that apply. You can also fill these out after creating
 - [ ] PR Description is included
 - [ ] `npm run vscode:prepublish` has been executed
 - [ ] All checks have passed (DCO, Jenkins and Code Coverage)
-- [ ] I have added tests (unit and integration) that prove my fix is effective or that my feature works
+- [ ] I have added unit tests that prove my fix is effective, that my feature works and it has test coverage
+- [ ] I have added integration tests that prove my fix is effective or that my feature works
+- [ ] I have tested it manually and there are no regressions found
 - [ ] I have added necessary documentation (if appropriate)
-- [ ] Any dependent changes have been merged and published in downstream modules
+- [ ] Any PR dependencies has been merged and published (if appropriate)
+
+## PR Reviewer Checklist
+
+_Put an `x` in the boxes that apply. If you're unsure about any of them, don't hesitate to ask. We're here to help!_
+
+- [ ] The Checklist above is complete before the PR is opened and ready for review.
+- [ ] The [CONTRIBUTOR GUIDANCE](https://github.com/zowe/vscode-extension-for-zowe/wiki/Best-Practices:-Contributor-Guidance) has been read and followed
+- [ ] PR title follows [Conventional Commits Guidelines](https://www.conventionalcommits.org/en/v1.0.0-beta.2/)
+- [ ] PR Description is included
+- [ ] `npm run vscode:prepublish` has been executed
+- [ ] All checks have passed (DCO, Jenkins and Code Coverage)
```

Comment:
Do we want to say here, "All tests and checks have passed"? (Mostly, I'm just trying to say "All unit and integration tests are passing", in addition to the other checks already listed in parentheses. Maybe that should be a separate list item, though.)

Reply:
@lauren-li The checks that I was talking about here are the ones that we have right now. Like CodeCov, DCO, Jenkins.

I have on separate checks for unit test and integration tests:

- [ ] Unit Tests has been added
- [ ] Integration Tests has been added
- [ ] There is coverage for the added code

Reply:
@jellypuno That makes sense. I was just thinking that even though there are list items for adding tests, there aren't any items for having the tests actually pass.

Reply:
aha. It was my assumption that when I say added, it also means passing. Because of the checks that we have. But I can add the word pass there